contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -232,7 +232,7 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
     uint64 offchainConfigVersion,
     bytes memory offchainConfig
   ) external override {
-    setConfig(
+    setConfigTypeSafe(
```

Comment:
This is probably a very stupid q, but just want to confirm that the onlyOwner modified will be applied on this call when called internally right?

Reply:
not stupid at all - and yes it still applies :)